### PR TITLE
More implementation

### PR DESCRIPTION
**Pull request by decriptor**

### CODE REVIEW

File: src/Gir/Model/Documentation.cs [hunk 1]
```diff
@@ -3,7 +3,6 @@
 
 namespace Gir
 {
-	[Serializable]
```
Comment (Therzok) [inline src/Gir/Model/Documentation.cs]: What?

File: src/Gir/Model/Include.cs [hunk 2]
```diff
@@ -3,7 +3,6 @@
 
 namespace Gir
 {
-	[Serializable]
```
Comment (Therzok) [inline src/Gir/Model/Include.cs]: Why remove the serializables here?

File: src/Gir/Utils.cs [hunk 3]
```diff
@@ -16,7 +16,7 @@ public static string ToCSharp (this string cname)
 
 			bool isUpper = true;
 			foreach (var c in cname) {
-				if (c == '_') {
+				if (c == '_' || c == '-') {
```
Comment (Therzok) [inline src/Gir/Utils.cs]: 👍

File: src/Gir/Model/Include.cs (hunk 2 above)
Comment (decriptor) [reply src/Gir/Model/Include.cs]: It was only on 3 files and doesn't appear to be needed. I was just going for consistency

**Pull request merged**